01-Setup-GreenSock-as-a-Module-with-Parcel.md

```diff
@@ -0,0 +1,56 @@
+# Setup GreenSock As A Module With Parcel
+
+📹 [Video](https://egghead.io/courses/create-amazing-animations-with-greensock)
```

Comment:
We can link to the course from the [README.md](https://github.com/eggheadio-projects/create-amazing-animations-with-greensock-notes/blob/master/README.md). Instead lets link to the lesson. 

For example: `https://egghead.io/lessons/greensock-setup-greensock-as-a-module-with-parcel`

Reply:
Let's do this for other lessons.